lib/file/widgets/choose_destination.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:black_hole_flutter/black_hole_flutter.dart';
+import 'package:flutter/material.dart';
+import 'package:schulcloud/app/app.dart';
+
+import '../data.dart';
+
+@immutable
+class FileDestination {
```

Comment:
Do we want to use this class throughout the whole module, maybe with a slightly different name (`path`)?